StructuralPatterns/Adapter/StringAdapter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Adapter
+{
+    public class StringAdapter : IStringAdapter
+    {
+        private string _inputString;
+
+        public StringAdapter()
+        {
+            _inputString = String.Empty;
+        }
+
+        public string InputString
+        {
+            get => _inputString;
+            set => _inputString = value;
```

Comment:
You could try ti get rid of the filed and make the property auto. Intitializing form the cinstructor should work fine as well.